core/app/assets/js/layout/header.js

```diff
@@ -0,0 +1,34 @@
+import HeaderButtons from './header-buttons';
+
+export default function Header( props ) {
+	React.useEffect( () => {
+		document.title = __( 'Elementor', 'elementor' ) + ' | ' + props.title;
+	} );
+
+	return (
+		<header className="elementor-app__header dialog-header dialog-lightbox-header">
```

Comment:
We have to decide about all classes prefixes

Reply:
Any new insights in regards to this ?

Reply:
done